src/core/lib/iomgr/polling_entity.h

```diff
@@ -22,6 +22,8 @@
 #include "src/core/lib/iomgr/pollset.h"
 #include "src/core/lib/iomgr/pollset_set.h"
 
+typedef enum pops_tag { POPS_NONE, POPS_POLLSET, POPS_POLLSET_SET } pops_tag;
```

Comment:
Hmm, this abbreviation isn't obvious to me. I think stylistically, non-obvious abbrevations shouldn't be used.

Reply:
I've just brought it outside the struct, since C++ does not allow using enum which are inside a struct.

Reply:
Ah, ok, this is uglier then. Since this now lives outside of a struct, it is polluting the global namespace. Anything in the global namespace needs to be prefixed `grpc_` . But I meant that the enum names are ugly.

Reply:
The grpc_ prefix is for functions as per https://github.com/grpc/grpc/blob/master/doc/c-style-guide.md. But it might make sense ti add I guess we can substitute it with
typedef enum pollset_tag {POLLS_NONE, POLLS_POLLSET, POLLS_POLLSET_SET} pollset_tag;
or
typedef enum grpc_pollset_tag {GRPC_POLLS_NONE, GRPC_POLLS_POLLSET, GRPC_POLLS_POLLSET_SET} grpc_pollset_tag;

Reply:
Then this is a defect in our C style guide. In practice, all the structs/enums declared in header files have the typename prefixed with `grpc_` and all the enum names are also prefixed with `GRPC_` . You should go ahead and fix this as `typedef enum grpc_pollset_tag ....` etc (your 2nd option above)

Reply:
Thanks Vijay! Done

Reply:
I have sent in PR #12519 to fix the style guide (as well as a few places where we weren't actually following the rules)